src/main/java/hudson/plugins/accurev/cmd/GetAccuRevVersion.java

```diff
@@ -0,0 +1,52 @@
+package hudson.plugins.accurev.cmd;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import org.apache.log4j.Logger;
+
+public class GetAccuRevVersion {
+
+    private static final Logger logger = Logger.getLogger(GetAccuRevVersion.class.getName());
+
+    /**
+     * Get current accuRev version by invoking the AccuRev CLI command
+     * 
+     * @return String accuRevVersion
+     */
+    public static String getAccuRevVersion() {
```

Comment:
In the current code base for every command there will be a separate file .Intially I add the same method in JustAccurev class which you deleted.